recipes/rmlui/3.3/conanfile.py

```diff
@@ -0,0 +1,100 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RmluiConan(ConanFile):
+    name = "rmlui"
+    description = "RmlUi - The HTML/CSS User Interface Library Evolved"
+    homepage = "https://github.com/mikke89/RmlUi"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "https://github.com/mikke89/RmlUi/blob/master/LICENSE"
+    topics = ("conan", "css", "gui", "html", "lua", "rmlui")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "build_lua_bindings": [True, False],
+        "build_samples": [True, False],
+        "disable_rtti_and_exceptions": [True, False],
+        "enable_precompiled_headers": [True, False],
+        "enable_tracy_profiling": [True, False],
+        "fPIC": [True, False],
+        "no_font_interface_default": [True, False],
+        "no_thirdparty_containers": [True, False],
+        "shared": [True, False]
+    }
+    default_options = {
+        "build_lua_bindings": False,
+        "build_samples": False,
+        "disable_rtti_and_exceptions": False,
+        "enable_precompiled_headers": True,
+        "enable_tracy_profiling": False,
+        "fPIC": True,
+        "no_font_interface_default": False,
+        "no_thirdparty_containers": False,
+        "shared": False
+    }
+    generators = "cmake_find_package"
+
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("freetype/2.10.1")
```

Comment:
seems optional depending on https://github.com/mikke89/RmlUi/blob/6e0e3011b23f617f1f301cee0dd595c7ae20e650/CMakeLists.txt#L185

Reply:
Thanks for noticing!